src/main/twirl/gitbucket/core/account/edit.scala.html

```diff
@@ -37,6 +37,11 @@
                 <input type="text" name="url" id="url" class="form-control" value="@account.url"/>
                 <span id="error-url" class="error"></span>
               </fieldset>
+              <fieldset class="form-group">
+                <label for="description" class="strong">bio (optional):</label>
```

Comment:
Update to `Bio`. I would like to unite the first character of labels as uppercase.